Source/Scene/Cesium3DTileset.js

```diff
@@ -121,7 +121,13 @@ define([
      * @param {Boolean} [options.debugShowBoundingVolume=false] For debugging only. When true, renders the bounding volume for each tile.
      * @param {Boolean} [options.debugShowContentBoundingVolume=false] For debugging only. When true, renders the bounding volume for each tile's content.
      * @param {Boolean} [options.debugShowViewerRequestVolume=false] For debugging only. When true, renders the viewer request volume for each tile.
-     * @param {Boolean} [options.debugShowGeometricError=false] For debugging only. When true, draws labels to indicate the geometric error of each tile
+     * @param {Boolean} [options.debugShowGeometricError=false] For debugging only. When true, draws labels to indicate the geometric error of each tile.
```

Comment:
These are probably too fine-grained.  Consider:

* Combine `debugShowNumberOfCommands`,  `debugShowNumberOfPoints`, `debugShowNumberOfTriangles`, and `debugShowNumberOfFeatures` into `debugShowRenderingStatistics`
* Combine `debugShowTextureMemoryUsage` and `debugShowVertexMemoryUsage` into `debugShowMemoryUsage`

This will be a lot less code and is inline with how I expect folks will want to use these hopefully without cluttering the display.

If clutter is a problem, try a smaller font that is still readable or a slightly less coarse-grained separation for the rendering stats.